templates/fstab.j2

```diff
@@ -6,8 +6,7 @@
 {%- if fstab_auto_root %}
 	{%- for mountpoint in ansible_mounts %}
 		{%- if mountpoint.mount=="/" %}
-			{%- set root={'options': mountpoint.options, 'source': mountpoint.device, 'type': mountpoint.fstype} %}
-			{{- fmtstr|format(root.source, '/', root.type, fstab_auto_root_options|default([root.options])|join(','),
+			{{- fmtstr|format(mountpoint.device, '/', fstab_auto_root_type|default(mountpoint.fstype), fstab_auto_root_options|default([mountpoint.options])|join(','),
```

Comment:
I think the list brackets in the default filter don't belong there. The variable already is a list. This way the filter creates a list of a list.

Reply:
mountpoint.device is option obtained by ansible setup and contains a string. It's exact the same behavior like bevor.

Reply:
Ah, yes. I only looked up `fstab_auto_root_options` in your readme, which is a list.